dizoo/cliffwalking/config/cliffwalking_dqn_config.py

```diff
@@ -0,0 +1,63 @@
+from easydict import EasyDict
+
+cliffwalking_dqn_config = dict(
+    exp_name='cliffwalking_dqn_seed0',
+    env=dict(
+        collector_env_num=8,
+        evaluator_env_num=8,
+        n_evaluator_episode=8,
+        stop_value=-13,
+        env_id='CliffWalking',
+        render_mode="rgb_array",
```

Comment:
we can omit this config field because it is usually the same as its default value